src/test/java/api/requests/RequestRulesTests.java

```diff
@@ -0,0 +1,40 @@
+package api.requests;
+
+import org.folio.circulation.domain.ItemStatus;
+import org.folio.circulation.domain.RequestRules;
+import org.folio.circulation.domain.RequestType;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class RequestRulesTests {
```

Comment:
I was thinking that the rules will change once we make them configurable per tenant, so the tests would break and would be meaningless then.  These tests were written to account for the logic of canCreateRequestForItem() rather than for the rules themselves. If you insist then I'd write tests for the rules.

Reply:
I’m not insisting. I likely have a different perspective on tests, I tend to see them as executable documentation.

I’m ok with these tests remaining this way, especially as I later noticed there were API tests for the specific page request behaviour.